code/modules/antagonists/changeling/powers/shriek.dm

```diff
@@ -1,10 +1,10 @@
 /datum/action/changeling/resonant_shriek
 	name = "Resonant Shriek"
-	desc = "Our lungs and vocal cords shift, allowing us to briefly emit a noise that deafens and confuses the weak-minded. Costs 20 chemicals."
+	desc = "Our lungs and vocal cords shift, allowing us to briefly emit a noise that deafens and confuses the weak-minded. Costs 30 chemicals."
 	helptext = "Emits a high-frequency sound that confuses and deafens humans, blows out nearby lights and overloads cyborg sensors."
 	button_icon_state = "resonant_shriek"
 	chemical_cost = 30
-	dna_cost = 1
+	dna_cost = 2

```

Comment:
I might be out of the loop, but has anyone actually been using resonant shriek?

Reply:
Oh, wait, you guys have that confusion wallbonking thing, don't you? I see why you're nerfing this, then.